src/jdk.httpserver/share/classes/com/sun/net/httpserver/Authenticator.java

```diff
@@ -71,11 +89,19 @@ public int getResponseCode() {
     public static class Success extends Result {
         private HttpPrincipal principal;
 
+        /**
+         * Creates a {@code Success} instance with given Principal.
```

Comment:
For consistency with line 103 below maybe Principal should be quoted `{@code Principal}` here and at line 95 too

Reply:
Changed to use {@code Principal}